src/main/scala/emm/effects/Binder.scala

```diff
@@ -29,6 +30,22 @@ trait BinderLowPriorityImplicits {
       NAP.pack(back)
     }
   }
+
+  implicit def binderState[S, C <: Effects](implicit C: Binder[C], T: Traverser[C], NN: NonNested[C]): Binder[State[S, ?] |: C] = new Binder[State[S, ?] |: C] {
```

Comment:
I need to go look at implementations, but thought: why does it make sense for the `run` of `StateT` to have `F` in both places. Namely, would `F[S => G[(S, A)]]` be (a) possible and (b) useful?